easybuild/easyconfigs/u/unblur/unblur-1.0.2.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'unblur'
+version = '1.0.2'
+
+homepage = 'http://grigoriefflab.janelia.org/unblur'
+description = """Unblur is used to align the frames of movies recorded on an electron microscope
+ to reduce image blurring due to beam-induced motion.
+ It reads stacks of movies that are stored in MRC/CCP4 format."""
+
+toolchain = SYSTEM
+
+sources = [{
+    'source_urls': ['https://grigoriefflab.umassmed.edu/system/'],
+    'filename': '%(name)s_%(version)s.tar.gz',
+    'download_filename': 'tdf?path=%(name)s_%(version)s.tar.gz&file=1&type=node&id=4902',
+}]
+
+checksums = ['1aa72b1f944114987ede644e1866eaebc08e191ecc566b3461409449360931e2']
+
+install_cmd = "cp -r %(namelower)s_%(version)s/bin %(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/unblur_openmp_7_17_15.exe',
+              'bin/unblur_plot_shifts.sh'],
```

Comment:
trying to run `unblur_plot_shifts.sh` gives me the error: `Gnuplot was not found on your system. Please make sure it is installed.`

Reply:
my bad, I've added Julia and gnu plot as dependencies